packages/react/package.json

```diff
@@ -21,6 +21,7 @@
     "node": ">=0.10.0"
   },
   "dependencies": {
+    "interaction-tracking": "16.4.3-alpha.0",
```

Comment:
Is the plan to unpin this? If this is pinned then there is a high risk of duplication.

Reply:
Ah, yeah. I'll change it to "^16.4.3-alpha.0"